src/repo/repository.rs

```diff
@@ -252,6 +252,19 @@ impl Repository {
         })
     }
 
+    pub fn packs_from_list(&self, mut reader: impl io::Read, separator: u8) -> Result<Vec<PackId>, Error> {
```

Comment:
This string processing would sit better in packs.rs (the CLI).

Reply:
hmm I'm having trouble moving this as it seems I can't get to the constructor of the `error::Error` enum there, it says it's a private type.  But because of the match the types of the branches need to be the same.

Reply:
oh.. nvm, figured it out :)